.eslintrc.js

```diff
@@ -33,5 +33,8 @@ module.exports = {
         "ts-check": "allow-with-description",
       },
     ],
+    "zustand-rules/use-store-selectors": "error",
+    "zustand-rules/no-state-mutation": "error",
+    "zustand-rules/enforce-use-setstate": "error",
```

Comment:
i can't get this to work but it's probably ok to not use this rule anyway?

the other two look good

Reply:
Discussed offline. The rule works, if you use `state.x = y`, but it doesn't enforce call to `set`